integration/spark/app/src/main/java/io/openlineage/spark/agent/EventEmitter.java

```diff
@@ -28,6 +28,16 @@ public class EventEmitter {
   @Getter private Optional<UUID> parentRunId;
 
   public EventEmitter(ArgumentParser argument) throws URISyntaxException {
+    this.jobNamespace = argument.getNamespace();
+    this.parentJobName = argument.getJobName();
+    this.parentRunId = convertToUUID(argument.getParentRunId());
+
+    if (argument.isConsoleMode()) {
+      this.client = new OpenLineageClient();
```

Comment:
Why this should construct console client? Why not create `ConsoleTransport` directly?

Reply:
ok, it's just default behavior of the client
```
 public OpenLineageClient() {
    this(new ConsoleTransport());
  }
```